webextensions/api/identity.json

```diff
@@ -29,7 +29,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/identity/getRedirectURL",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "33"
```

Comment:
✅ https://github.com/chromium/chromium/commit/aee340ad704c00a9b2dafe52353a5e55bb1f9e09 landed in [33.0.1704.0](https://github.com/chromium/chromium/releases/tag/33.0.1704.0) before 33.0.1750.